### DISCUSSION TITLE
Not able to login to Budibase cloud

### DISCUSSION THREAD

**Discussion by rishav-propertyguru** [Budibase/budibase#13758] [Help] [answered]
Hi Everyone,
I used to login to my Budibase client account using Entra SSO, and i had the password saved in a password manager.
While configuring the SSO on Budibase, i tried logging out and logging in with the new SSO.

But now i am logged out of my account, also my password it showing wrong. Reset password is showing "Password reset email sent" but i am not receiving any email.

Need urgent help on this.

**Comment by mikesealey** [comment]
Hi @rishav-propertyguru. I'm sorry to hear you're having issues. What method of self-host are you using? Docker-compose/CLI, etc

**Reply by rishav-propertyguru** [reply]
Hi @mikesealey  it the Budibase cloud not the self hosted, my domain is [masked]

**Reply by mikesealey** [reply]
Hi @rishav-propertyguru, I've passed this on to a member of the Budibase team who was able to take a look at what's going on behind the scenes.

For an unknown reason the password field seems to have been missed, and a randomly generated one has been made on your behalf. Please try to recover your password again and check your junk/spam folders for the recovery email, hopefully it's just been incorrectly flagged as spam

**Reply by rishav-propertyguru** [reply]
Thank you so much @mikesealey for taking this to Budibase, but i am still not getting the recovery email, i am checking my spam too.
Can you please request them to send me a random password on [masked]

**Reply by rishav-propertyguru** [reply]
hi @mikesealey i really need you help here.

**Comment by melohagan** [comment]
Hey @rishav-propertyguru 

How did you create your account initially? Did you sign up with the same SSO, or did you sign up with email/password or a different SSO account?

Could you please share a video to demonstrate the problem that you are seeing, and I can try to recreate.

**Reply by rishav-propertyguru** [reply]
HI @melohagan, i created the account using email & password, and later integrated SSO, after integrating sso, i was using SSO to login.
This time i was replacing the older SSO with a Different SSO i did some SSO configuration mistake.
Now i am logged out, i am not able to sign in using email & password neither i am able to reset my password also SSO is not working due to configuration issue.

this is my subdomain [masked]

**Reply by rishav-propertyguru** [reply]
Here's the recordings @melohagan 

[Removed]

**Reply by melohagan** [reply]
Thanks @rishav-propertyguru this was very helpful.

I have reset your account password, could you please check your email?

**Reply by rishav-propertyguru** [reply]
I have received an email but it's giving me same issue as the 2nd loom video attached @melohagan

**Reply by rishav-propertyguru** [reply]
`{
    "message": "Password change is disabled for this user",
    "status": 400
}`

**Reply by melohagan** [reply]
Okay, the same error as before. One option is that we could delete the tenant, but otherwise I can look into changing your account type/unlocking it in the backend if you'd prefer that.

**Reply by rishav-propertyguru** [reply]
We Don't have the backup of any data.
If there's a way to recover my access without losing the data & the dashboards i have built, kindly suggest

**Comment by rishav-propertyguru** [comment, marked as answer]
Fortunately my manager was added as one of the admins, i got the access back.
Thanks everyone for the help.

**Reply by melohagan** [reply]
Fantastic, great to hear! I have taken this scenario onboard however and will look to fix the issues for future users.

**Reply by rishav-propertyguru** [reply]
i figure out that SMTP configuration is not working, i have provided right information yet neither it's throwing an error while resetting the password nor sending out the email

**Reply by melohagan** [reply]
Hey @rishav-propertyguru I was recently using the Send Email action in Budibase Cloud using Mailgun. Double check your config is correct; feel free to share the error you are seeing
